# portfolioLinks

* **Links to software I have developed for other organisations or GitHub repos:**

  * https://github.com/CGEBM/BarSeq (A *wrapper* script for NGS Bar-seq analysis)
  * https://github.com/acbellorib/DIO-Everis_FSDev_Tasks (Tasks carried out while attending the DIO-Everis Full Stack Developer Bootcamp (https://digitalinnovation.one/bootcamps/everis-fullstack-developer))
  
* **Links to my peer-reviewed scientific publications (main authorship and co-authorship):**

  * https://bmcresnotes.biomedcentral.com/articles/10.1186/1756-0500-7-132 (STINGRAY: system for integrated genomic resources and analysis)
  * https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0801-z (An investigation of causes of false positive single nucleotide polymorphisms using simulated reads from a small eukaryote genome)
  * https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1006405 (Elevated catalase expression in a fungal pathogen is a double-edged sword of iron)
  * https://link.springer.com/article/10.1007%2Fs10681-019-2427-0 (QTL-seq reveals a major root-knot nematode resistance locus on chromosome 11 in rice (*Oryza sativa* L.))
  
* **Links to my M.Sc. dissertation and Ph.D. thesis:**

  * https://www.arca.fiocruz.br/handle/icict/6943 (LASZLO @ GALAXY - Um protótipo de serviço de montagem de genomas a partir de dados de sequenciamento de próxima geração (NGS)**)
  * https://discovery.dundee.ac.uk/en/studentTheses/a-quantitative-exploration-of-causes-of-false-positive-single-nuc (A Quantitative Exploration of Causes of False Positive Single Nucleotide Polymorphisms in Next-Generation Sequencing Data)
  
* **My LinkedIn profile:**

  * https://www.linkedin.com/in/acbr-2a54b61/

